### ISSUE TITLE
Please!RuntimeError:BatchNorm: expected Variable at argument 0 (got None)

### ISSUE THREAD

**Issue by ztttkx0804** [andreasveit/densenet-pytorch#7]
when I runed the same code with you ?How to solve this error?Thank you very much!
I have solved the last question,Thank you very much

**Comment by kiddding** [comment]
Did you miss the 'return output'?
